docs/installation/linux/centos.md

```diff
@@ -29,101 +29,139 @@ your kernel must be 3.10 at minimum, which CentOS 7 runs.
-        enabled=1
-        gpgcheck=1
-        gpgkey=https://yum.dockerproject.org/gpg
-        EOF
+    ```bash
+    $ sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
+    [dockerrepo]
+    name=Docker Repository
+    baseurl=https://yum.dockerproject.org/repo/main/centos/7/
+    enabled=1
+    gpgcheck=1
+    gpgkey=https://yum.dockerproject.org/gpg
+    EOF
+    ```
 
 4. Install the Docker package.
 
-        $ sudo yum install docker-engine
+    ```bash
+    $ sudo yum install docker-engine
+    ```
 
-5. Start the Docker daemon.
+5. Enable the service.
 
-        $ sudo service docker start
+    ```bash
+    $ sudo systemctl enable docker.service
+    ```
 
-6. Verify `docker` is installed correctly by running a test image in a container.
+6. Start the Docker daemon.
+
+    ```bash
+    $ sudo systemctl start docker
+    ```
+
+7. Verify `docker` is installed correctly by running a test image in a container.
+
+        $ sudo docker run --rm hello-world
 
-        $ sudo docker run hello-world
         Unable to find image 'hello-world:latest' locally
-    		latest: Pulling from hello-world
-    		a8219747be10: Pull complete
-    		91c95931e552: Already exists
-    		hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
-    		Digest: sha256:aa03e5d0d5553b4c3473e89c8619cf79df368babd1.7.1cf5daeb82aab55838d
-    		Status: Downloaded newer image for hello-world:latest
-    		Hello from Docker.
-    		This message shows that your installation appears to be working correctly.
-
-    		To generate this message, Docker took the following steps:
-    		 1. The Docker client contacted the Docker daemon.
-    		 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
-    				(Assuming it was not already locally available.)
-    		 3. The Docker daemon created a new container from that image which runs the
-    				executable that produces the output you are currently reading.
-    		 4. The Docker daemon streamed that output to the Docker client, which sent it
-    				to your terminal.
-
-    		To try something more ambitious, you can run an Ubuntu container with:
-    		 $ docker run -it ubuntu bash
-
-    		For more examples and ideas, visit:
-    		 http://docs.docker.com/userguide/
+        latest: Pulling from library/hello-world
+        c04b14da8d14: Pull complete
+        Digest: sha256:0256e8a36e2070f7bf2d0b0763dbabdd67798512411de4cdcf9431a1feb60fd9
+        Status: Downloaded newer image for hello-world:latest
 
-### Install with the script
+        Hello from Docker!
+        This message shows that your installation appears to be working correctly.
+
+        To generate this message, Docker took the following steps:
+         1. The Docker client contacted the Docker daemon.
+         2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
```

Comment:
nit: Using actual numbers in a Markdown numbered list is ineffective and can be confusing if the ordering changes later. Consider just using "1." for all of them.